This file is part of MXE. See LICENSE.md for licensing information.

Contains ad hoc patches for cross building.

From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Robert Manner <robert.manner@balabit.com>
Date: Fri, 15 Dec 2017 16:40:18 +0100
Subject: [PATCH] scons/crossmingw.py: compiler prefix can be customized
 through environment


diff --git a/scons/crossmingw.py b/scons/crossmingw.py
index 1111111..2222222 100644
--- a/scons/crossmingw.py
+++ b/scons/crossmingw.py
@@ -51,6 +51,9 @@ prefixes64 = SCons.Util.Split("""
 """)
 
 def find(env):
+    if os.environ['MINGW_PREFIX']:
+        return os.environ['MINGW_PREFIX']
+
     if env['machine'] == 'x86_64':
         prefixes = prefixes64
     else:
